lib/Model/Listing/AbstractListing.php

```diff
@@ -411,8 +411,14 @@ public function setGroupBy($groupBy, $qoute = true)
         if ($groupBy) {
             $this->groupBy = $groupBy;
 
-            if ($qoute && strpos($groupBy, '`') !== 0) {
-                $this->groupBy = '`' . $this->groupBy . '`';
+          if ($qoute && !str_starts_with($groupBy, '`')) {
```

Comment:
I'm wondering whether it would make sense to just replace all occurrences of  \`. 
Otherwise it would be still possible to add unquoted stuff, even when `$quote = true`. WDYT?

Reply:
Replacing it with what? `quoteIdentifier` escapes the backtick signs, if there are any. So basically it already does a replace.